### PR TITLE
feat(prices): add MDB and MDB+

### PR DESCRIPTION
**Pull request by RomualdH**
- Add MDB and MDB+ on BNB Chain
- ~~Add SPHERE on Polygon~~ Will be done later as the address of the token should be updated on Coin Paprika.

**For Dune Engine V2**

I've checked that:

### General checks:
* [x] I tested the query on dune.com after compiling the model with dbt compile (compiled queries are written to the target directory)

### Pricing checks:
* [x] `coin_id` represents the ID of the coin on coinpaprika.com
* [x] all the coins are active on coinpaprika.com (please remove inactive ones)

### CODE REVIEW

File: models/prices/prices_tokens.sql [hunk 1]
```diff
@@ -1776,6 +1778,7 @@ VALUES
     ("nuls-nuls", "polygon", "NULS", "0x8b8e48a8cc52389cd16a162e5d8bd514fabf4ba0", 8),
     ("blank-blockwallet","polygon", "BLANK", "0xf4C83080E80AE530d6f8180572cBbf1Ac9D5d435", 18),
     ("fin-fin", "polygon", "FIN", "0x576c990a8a3e7217122e9973b2230a3be9678e94", 18),
+    -- ("sphere-sphere-finance", "polygon", "SPHERE", "0x62f594339830b90ae4c084ae7d223ffafd9658a7", 18),
```
Comment (Hosuke) [inline models/prices/prices_tokens.sql:1781]: https://api.coinpaprika.com/v1/coins/sphere-sphere-finance
This token is not active, so comment out.

File: models/prices/prices_tokens.sql [hunk 2]
```diff
@@ -333,6 +333,8 @@ VALUES
     ("xen-xen-crypto", "bnb", "XEN", "0x2AB0e9e4eE70FFf1fB9D67031E44F6410170d00e", 18),
     ("bob-bob", "bnb", "BOB", "0xb0b195aefa3650a6908f15cdac7d92f8a5791b0b", 18),
     ("fb-fbbank", "bnb", "FB", "0x770F030FDBf63EBF1C939De8BCFf8943C2C2d454", 2),
+    ("mdb-mdb", "bnb", "MDB+", "0x9f8bb16f49393eea4331a39b69071759e54e16ea", 18),
+    ("mdb-milliondollarbaby", "bnb", "MDB", "0x0557a288a93ed0df218785f2787dac1cd077f8f3", 18),
```
Comment (Hosuke) [inline models/prices/prices_tokens.sql:337]: https://api.coinpaprika.com/v1/coins/mdb-mdb
https://api.coinpaprika.com/v1/coins/mdb-milliondollarbaby

**Review by Hosuke — APPROVED** [review]
LGTM.✅
Thank you @RomualdH

File: models/prices/prices_tokens.sql (hunk 1 above)
Comment (RomualdH) [reply models/prices/prices_tokens.sql:1781]: Hey @Hosuke, do you expect something from me?

**Review by soispoke — APPROVED** [review]

File: models/prices/prices_tokens.sql (hunk 1 above)
Comment (Hosuke) [reply models/prices/prices_tokens.sql:1781]: No, just explain why this line is in comment, as this token is no longer active in api.

Comment (Hosuke) [reply models/prices/prices_tokens.sql:1781]: No, just explain why this line is in comment, as this token is no longer active in api.

Comment (Hosuke) [reply models/prices/prices_tokens.sql:1781]: Sorry for the broken English.

**Pull request merged**